lib/embulk/input/google_analytics/plugin.rb

```diff
@@ -102,8 +102,8 @@ def run
 
           latest_time_series = nil
           client.each_report_row do |row|
-            time = row[task["time_series"]]
-            next if last_record_time && time <= last_record_time
+            time = row["ga:dateHour"] ? row["ga:dateHour"] : row["ga:date"]
+            next if last_record_time && time.utc <= last_record_time.utc
```

Comment:
Hmm. I can't understand why adding `.utc` fixed the problem. Ruby compares times with time zone offset as below.

```
$ irb -r time
irb(main):001:0> t = Time.now
=> 2017-03-27 14:37:47 +0900
irb(main):002:0> t.utc
=> 2017-03-27 05:37:47 UTC
irb(main):003:0> t.utc < t
=> false
irb(main):004:0> t.utc <= t
=> true
irb(main):005:0> t.utc == t
=> true
```